packages/unsafe-eval/src/install.ts

```diff
@@ -6,52 +6,33 @@ interface PIXICore {
     ShaderSystem: typeof ShaderSystem;
 }
 
-export function install(PIXI: PIXICore): void
+export function install({ ShaderSystem }: PIXICore): void
```

Comment:
If you want people to be able to do `install({ ShaderSystem })` as in the readme, you'll have to change the argument type here from `: PIXICore` to `: { ShaderSystem: typeof ShaderSystem }` (or something similar, if that exact syntax is incorrect).

Reply:
Nevermind, I expanded the code to see it is already that way.